server/modules/authentication/oauth2/authentication.js

```diff
@@ -37,7 +37,13 @@ module.exports = {
     })
 
     client.userProfile = function (accesstoken, done) {
-      this._oauth2._useAuthorizationHeaderForGET = true
+      const url = new URL(conf.userInfoURL)
+      if (conf.useAuthorizationHeader) {
+        this._oauth2._useAuthorizationHeaderForGET = true
+      } else {
+        this._oauth2._useAuthorizationHeaderForGET = false
+        url.searchParams.set('access_token', accesstoken)
```

Comment:
You're creating an `url` variable, set a searchParams but then it's never used again? Am I missing something?

Also the current default would cause existing installations to suddenly use the opposite behavior. Either set the useAuthorizationHeader default as true or reverse the logic...

Reply:
Completely forgot about the url variable. Will reverse the logic. Thx for the review gonna mend the things and test them again.